docs/docs/overview.md

```diff
@@ -51,9 +51,9 @@ By default port `8080` is used. A custom port can be used via the `--port` optio
 :::
 
 :::tip
-It's also possible to set a custom port for the dart vm service using `--dart-vm-service-port` option.
+It's also possible to set a custom port for the Dart VM service using the `--dart-vm-service-port` argument.
 
-This is required when trying to run `dart_frog dev` multiple times.
+This is required when trying to run multiple `dart_frog dev` simultaneously on the same host.
```

Comment:
```suggestion
This is required when trying to run multiple `dart_frog dev` simultaneously on the same host.
```

we're missing a noun here. is it "...when trying to run multiple `dart_frog dev` commands simultaneously..." or something else?

Reply:
I was treating the `dart_frog dev` as the noun but the above makes sense.